src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook_test.go

```diff
@@ -544,7 +544,7 @@ func TestValidateUpdate_NoError(t *testing.T) {
 	})
 }
 
-//nolint:funlen // this is ok for a test
+//nolint:funlen,dupl // this is ok for a test
```

Comment:
this test is getting out of proportion, do you think we can separate the new tests into its own methods with _SchemaRegistry suffix?